charts/datadog/templates/_container-system-probe.yaml

```diff
@@ -78,6 +78,43 @@
       mountPath: /etc/datadog-agent/runtime-security.d
       readOnly: true
 {{- end }}
+{{- if eq (include "runtime-compilation-enabled" .) "true" }}
+    - name: runtime-compiler-output-dir
+      mountPath: {{ .Values.datadog.systemProbe.runtimeCompilationAssetDir }}/build
+      mountPropagation: {{ .Values.datadog.hostVolumeMountPropagation }}
+    - name: kernel-headers-download-dir
+      mountPath: {{ .Values.datadog.systemProbe.runtimeCompilationAssetDir }}/kernel-headers
+      readOnly: false
+{{- if .Values.datadog.systemProbe.mountPackageManagementDirs }}
+    - name: etc-os-release
+      mountPath: /host/etc/os-release
+      readOnly: true
+    - name: etc-redhat-release
+      mountPath: /host/etc/redhat-release
+      readOnly: true
+    - name: etc-fedora-release
+      mountPath: /host/etc/fedora-release
+      readOnly: true
+    - name: etc-lsb-release
+      mountPath: /host/etc/lsb-release
+      readOnly: true
+    - name: apt-config-dir
+      mountPath: /host/etc/apt
+      readOnly: true
+    - name: yum-repos-dir
+      mountPath: /host/etc/yum.repos.d
+      readOnly: true
+    - name: opensuse-repos-dir
+      mountPath: /host/etc/zypp
+      readOnly: true
+    - name: public-key-dir
+      mountPath: /etc/pki
+      readOnly: true
+    - name: rhel-subscription-dir
+      mountPath: /etc/rhsm
+      readOnly: true
```

Comment:
are these intentionally not mounted at `/host/` ?

Reply:
No, they should be under there as well. Good catch :)